serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,164 @@
++
+--
+However, the `pom.xml` file contains the following resource:
+
+.Example `pom.xml` file
+[source,xml]
+----
+<resource>
+ <directory>${project.basedir}/../serverless-workflow-greeting-server-rpc-quarkus/src/main/proto</directory>
+ <includes>
+   <include>greeting.proto</include>
+  </includes>
+</resource>
+----
+
+Note that in the gRPC proto file, the specified URI does not contain any scheme, therefore the file is expected to be accessible in the classpath of the project. The previous example of `pom.xml` file ensures that the proto file defined in the gRPC project is included in the classpath of the Serverless Workflow application.
+
+Either the proto file is accessible in the project directory structure, in which you manually copy the proto file in a directory that is included by default in the classpath, or the proto file can be downloaded from a remote server, in which the URI includes the `http` scheme. The `file` scheme covers the cases in which proto files are stored in a shared directory and the full path of the directory is known by the Serverless Workflow application.
+--
+
+* link:https://github.com/kiegroup/kogito-examples/blob/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-client-rpc-quarkus/src/main/resources/application.properties[Application properties]: In the `application.properties` file, you can add the information of gRPC location for the Serverless Workflow application to locate.
++
+--
+.Example `application.properties` file
+[source,properties]
+----
+quarkus.grpc.clients.Greeter.host=localhost
+quarkus.grpc.clients.Greeter.port=50051
+----
+
+Internally, the Serverless Workflow implementation uses the same set of properties as link:https://quarkus.io/guides/grpc-service-consumption#client-configuration[Quarkus gRPC client]. The `client-name` used by Kogito is the service name declared in the proto file, such as `Greeter`.
+
+In the same GitHub repository as the example application, a link:https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-server-rpc-quarkus[maven project], provides a simple implementation of the `Greeter` service. By default, this gRPC server runs on port `50051`. Therefore, the same port is used in the `application.properties` file.
+--
+
+You can also use the example application `serverless-workflow-greeting-client-rpc-quarkus` in
```

Comment:
I would maybe clarify that it is in fact the "complete" SWF application, when I was reading it it wasn't obvious until I clicked on the link.

Reply:
this parragraph does not really meke sense, I removed it